executor/set.go

```diff
@@ -282,6 +282,11 @@ func (e *SetExecutor) getVarValue(v *expression.VarAssignment, sysVar *variable.
 		if sysVar != nil {
 			return sysVar.Value, nil
 		}
+		// Setting timestamp to DEFAULT causes its value to be the current date and time as of the time it is accessed.
+		// See https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_timestamp
+		if v.Name == variable.Timestamp {
+			return "default", nil
```

Comment:
If you could find a constant for "default" please use it, otherwise OK as is :)

Reply:
> If you could find a constant for "default" please use it, otherwise OK as is :)

Seems not to find any constant for "default", the code below, use "default" as well
https://github.com/pingcap/tidb/blob/0192002f41e3d580dca46ba6ab4c01df9a09780d/sessionctx/variable/sysvar.go#L283

Reply:
Is it possible to remove this code? Ideally we should not add special handling for sysvars in `executor/set.go`, if it can be handled in the `GetSession` func.